app/facades/food_data_facade.rb

```diff
@@ -0,0 +1,14 @@
+class FoodDataFacade
+
+  def self.foods(query)
+    food_details = FoodDataService.search_foods(query)[:foods][0..9]
```

Comment:
Nice job only taking the first 10 food itemsfrom these results. This way you only ever create 10 FoodDataDetails objects.  You could also have passed a `pageSize` query param to the API get request to ensure only 10 come back, which would likely speed this up.